janusgraph-core/src/main/java/org/janusgraph/graphdb/configuration/JanusGraphConstants.java

```diff
@@ -60,6 +55,18 @@
      */
     public static final String TITAN_ID_STORE_NAME = "titan_ids";
 
+
+    /**
+     * Storage format version currently used by JanusGraph, version 1 is for JanusGraph 0.2.x and below
+     */
+    public static final String STORAGE_VERSION;
+
+
+    /**
+     * List of FIXED fields that can be modified when graph.allow-upgrade is set to true 
+     */
+    public static final List<String> UPGRADEABLE_FIXED;
```

Comment:
As I understand, it should be `Set` and not `List`.

Reply:
Thanks for the review. I was trying to avoid adding any additional imports, but I agree a set would perform slightly better. I applied all the changes you suggested and will push them once I do a test upgrade locally.